exec/csnex/Cell.cs

```diff
@@ -38,5 +93,50 @@ public Cell(String s)
             String = s;
             type = Type.String;
         }
+
+        public Cell(Number d)
+        {
+            Number = d;
+            type = Type.Number;
+        }
+#endregion
+
+        public void ResetCell()
+        {
+            Address = null;
+            Number = null;
+            Object = null;
+            String = null;
+            type = Type.None;
+        }
+
+        public void Set(Cell c)
+        {
+            if (type == Type.None)
+            {
+                type = Type.Address;
+            }
+            Debug.Assert(type == Type.Address);
+            Address = c;
+        }
+
+        public void Set(Number n)
+        {
+            if (type == Type.None) {
```

Comment:
Please be consistent about where place your braces. I don't care whether you decide to put them on the end of the line (like here) or on a line by themselves (next function), but the most important thing is to be *consistent*.

Reply:
Opps.  Sorry.  That was Visual Studio trying to read my mind, I guess.